src/design/optForce/optForce.m

```diff
@@ -0,0 +1,1000 @@
+function [optForceSets, posOptForceSets, typeRegOptForceSets, flux_optForceSets] = optForce(model,...
+    targetRxn, mustU, mustL, minFluxesW, maxFluxesW, minFluxesM, maxFluxesM, k,...
+    nSets, constrOpt, excludedRxns, runID, outputFolder, outputFileName,... 
+    printExcel, printText, printReport, keepInputs, verbose)
+% This function runs the third step of optForce that is to solve a
+% bilevel mixed integer linear programming problem to find sets of
+% interventions that lead to an increased production of a particular target
+%
+%
+% USAGE: 
+%
```

Comment:
If possible, with this many optional inputs, I would really recommend an [input parser](https://nl.mathworks.com/help/matlab/ref/inputparser-class.html), otherwise this is extremely prone to typos. e.g. when a user wants a verbose output he needs to give inputs for everything else.